TotallyNotMonopolyAPI/Services/PlayerService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Model;
+using System;
+
+namespace Services
+{
+    public class PlayerService
+    {
+
+        public Player[] CreateRamdonPlayers()
+        {
+            Player[] players = new Player[4];
+            
+            Player player1 = new Player();
+            player1.Id = 1;
+            player1.Money = 50000;
+            player1.Name = "Martin";
+
+            players[0] = player1;
+
+            Player player2 = new Player();
+            player2.Id = 2;
+            player2.Money = 50000;
+            player2.Name = "Marcell";
+
+            players[1] = player2;
+
+            Player player3 = new Player();
+            player1.Id = 3;
+            player1.Money = 50000;
+            player1.Name = "Daniel";
+
+            players[2] = player3;
+
+            Player player4 = new Player();
+            player4.Id = 4;
+            player4.Money = 50000;
+            player4.Name = "Robert";
+
+            players[3] = player4;
+
+            return players;
+        }
+
+        public Player BuyProperty(Player player, Property property)
+        {
+            if (player.Money >= property.cost)
+            {
+                player.properties.Add(property);
+            }
+            else
+            {
+                return player;
+            }
+
+            return player;
+        }
+
+        public Player SellProperty(Player player, Property property)
+        {
+            property.Owner = null;
+            player.properties.Remove(property);
+
+            return player;
+        }
+
+        public Player PayRent(Player playerOwner, Player playerTenant, Property property)
+        {
+            // To Do
```

Comment:
Try to avoid todo-s. Leave this one I'll implement the needed stuff for it and fill this too.